hermes-management/src/main/java/pl/allegro/tech/hermes/management/api/OfflineReadersEndpoint.java

```diff
@@ -0,0 +1,44 @@
+package pl.allegro.tech.hermes.management.api;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import pl.allegro.tech.hermes.management.domain.readers.OfflineReadersService;
+
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.Response;
+
+import static javax.ws.rs.core.MediaType.APPLICATION_JSON;
+
+@Component
+@Path("/readers")
+public class OfflineReadersEndpoint {
+
+    private static final Logger logger = LoggerFactory.getLogger(OfflineReadersEndpoint.class);
+
+    private final OfflineReadersService offlineReadersService;
+
+    @Autowired(required = false)
+    OfflineReadersEndpoint(OfflineReadersService offlineReadersService) {
+        if (offlineReadersService == null) {
+            logger.info("Offline readers bean is absent");
+        }
+        this.offlineReadersService = offlineReadersService;
+    }
+
+    @GET
+    @Path("/{topic}")
```

Comment:
We already have a path `/topics/<topic>` with topic details, I would put the readers endpoint under that namespace, i.e. `/topics/<topic>/...`.